site/source/docs/porting/emterpreter.rst

```diff
@@ -24,6 +24,8 @@ You can optionally use ``-s 'EMTERPRETIFY_FILE="data.binary"'`` to store the emt
 
 As usual, you can grep the ``tests/`` folder for examples of emterpreter usage in the test suite (search for ``EMTERPRETIFY``).
 
+    .. note:: setting ``EMTERPRETIFY=1`` forces ``js-opts=1``.
```

Comment:
please add something like "`, because we convert to the emterpreter's binary format using a js-optimizer pass`."